lua/codeium/api.lua

```diff
@@ -442,8 +442,8 @@ function Server:new()
 			cursor_offset = 0,
 			text = text,
 			line_ending = line_ending,
-			absolute_path = vim.fn.fnamemodify(vim.api.nvim_buf_get_name(bufnr), ":p"),
-			relative_path = vim.fn.fnamemodify(vim.api.nvim_buf_get_name(bufnr), ":"),
+			absolute_uri = util.get_uri(vim.api.nvim_buf_get_name(bufnr)),
+			workspace_uri = util.get_uri(util.get_relative_path(bufnr)),
```

Comment:
```suggestion
			relative_path_migrate_me_to_workspace_uri = util.get_uri(util.get_relative_path(bufnr)),
```

Reply:
This didn't quite work in my testing as written, but I just pushed something similar which used `relative_path_migrate_me_to_workspace_uri`. Thanks!

Reply:
Maybe it would work when combined with the changes in #247?

Reply:
I think so yes. The change I made here works properly when the "workspace root" is the CWD (which it is in this version of the code). But yeah the proper fix will be in #247. 

I'm in favor of just merging this as-is since it's an improvement and then #247 once it's ready, just to get people unblocked and able to use Chat again. But we can also just close this and do 247 instead if you prefer.